src/pages/api/cardholders.ts

```diff
@@ -106,11 +119,12 @@ const createCardholder = async (req: NextApiRequest, res: NextApiResponse) => {
 
 const updateCardholder = async (req: NextApiRequest, res: NextApiResponse) => {
   const session = await getSessionForServerSide(req, res);
-  const StripeAccountId = session.accountId;
+  const { stripeAccount } = session;
+  const { accountId, platform } = stripeAccount;
 
   const ip =
     req.headers["x-real-ip"]?.toString() || req.connection.remoteAddress;
-  const stripe = stripeClient();
+  const stripe = stripeClient(platform);
```

Comment:
a not-insignificant portion of this _large_ diff is just doing this over and over again (replacing the stripe account ID string that we pass around with a `StripeAccount` data object, which is basically a pairing of the stripe account ID with some identifier for its parent platform - which is now necessary with multiple platforms in play)